tables/Tables/Table.h

```diff
@@ -1003,33 +1003,33 @@ friend class TableExprNodeRep;
     // Some deprecated functions for backward compatibility, now in TableUtil.h.
     // Use old way of indicating deprecate to avoid -Wc++14-extensions warnings.
     // <group>
-     // [[deprecated ("Now use TableUtil::openTable")]]
+     //# [[deprecated ("Now use TableUtil::openTable")]]
```

Comment:
Why did you add the # ? I don't mind, but I couldn't find any documentation what # does here.

Reply:
//# lines are not included in the doxygen documentation.
casacore used to have its own documentation processor, but switched to doxygen quite some time ago. A preprocessing script turns comment lines with casacore tags into doxygen lines.